<launch>

  <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="frame_grabber" args="load sl_sensor_frame_grabber/FrameGrabberNodelet standalone_nodelet" output="screen">
    <param name="image_topic" type="string" value="/path1" />
    <param name="projector_topic" type="string" value="/path2" />
    <param name="output_topic" type="string" value="/path3" />
    <param name="frame_id" type="string" value="/path4" />
    <param name="lower_bound_tol" type="double" value="0.008" />
    <param name="upper_bound_tol" type="double" value="0.008" />
    <param name="image_trigger_period" type="double" value="0.016666667" />
  </node>		  

</launch>